server/controllers/stock/reports/stock_exit_loss.receipt.handlebars

```diff
@@ -19,8 +19,10 @@
           <span class="text-uppercase">
             {{translate 'STOCK.RECEIPT.EXIT_LOSS'}}
           </span> <br>
-          <span>{{details.document_reference}}</span> <br>
-          <small>{{date details.date}}</small> <br>
+          <div class="row">
+            <small>{{> barcode value=details.barcode}}</small>
+            <br/><br/>
```

Comment:
Could you remove this `<br/><br/>`, i think it is not necessary to have these spaces